web_programming/instagram_crawler.py

```diff
@@ -102,10 +102,10 @@ def test_instagram_user(username: str = "github") -> None:
     A self running doctest
     >>> test_instagram_user()
     """
-    from os import getenv
+    import os
 
-    if getenv("CONTINUOUS_INTEGRATION"):
-        return  # test failing on Travis CI
+    if os.environ.get("GITHUB_ACTIONS"):
```

Comment:
```suggestion
    if os.environ.get("CI"):
```
They all seem to do `CI` so let's remain flexible.  https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables

Reply:
GitHub suggests otherwise:

> GITHUB_ACTIONS | Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.

Reply:
I do not think that it is suggesting differently.  I like the description next to CI even more. ;-)  It is always set to True.  I trust that more.  CI is supported by Travis CI, CircleCI, and GHA so it is less fragile than hard coding GHA.

Also, why the change https://stackoverflow.com/questions/10952507/when-would-os-environfoo-not-match-os-getenvfoo and https://stackoverflow.com/questions/16924471/difference-between-os-getenv-and-os-environ-get

Reply:
Alright, also take a look at https://stackoverflow.com/a/55940480

One more reason is habit. I was working on a feature in https://github.com/brettcannon/gidgethub where `os.environ` was used everywhere. 

`getenv` is just a wrapper around `environ`